lib/src/cfi.rs

```diff
@@ -187,7 +187,7 @@ pub fn cfi_assert_eq_12_words(a: &[u32; 12], b: &[u32; 12]) {
         core::arch::asm!(
             "j 3f",
             "2:",
-            "li a0, 0x1040055",
+            "li a0, 0x04",
```

Comment:
Can you add a comment here (and elsewhere in the embedded assembly) that this maps to `CfiPanicInfo::AssertEqFail`? Or if the `asm!` macro supports passing constants from the Rust code, that would also be clear.